### ISSUE TITLE
Test against WordPress 5.8

### ISSUE THREAD

**Issue by jeffpaul** [10up/classifai#261]
**Is your enhancement related to a problem? Please describe.**
Once WordPress 5.8 is released, we'll want to test ClassifAI to see if any incompatibility issues arise.

**Describe the solution you'd like**
- [x] test ClassifAI on WordPress 5.8
- [ ] open issues for any incompatibilities noted in testing
- [ ] resolve issues identified in testing
- [x] bump "tested up to" version
- [ ] if code changes needed due to incompatibilities, ship a plugin release

**Designs**
n/a

**Describe alternatives you've considered**
none

**Additional context**
Related: #205.

**Comment by phpbits** [comment]
@jeffpaul Found this issue when using version Classifai 1.6.0. Errors are only visible when on Gutenberg editor. Thanks!

<img width="1325" alt="Screen Shot 2020-12-23 at 3 58 22 AM" src="https://user-images.githubusercontent.com/3365507/102928095-1bcadd00-44d3-11eb-9977-8acec70b93f2.png">

**Comment by phpbits** [comment]
@jeffpaul This PR fixes the error but I'm not sure what's the implication with the REST API calls : https://github.com/10up/classifai/pull/263. Thanks!

**Comment by jeffpaul** [comment]
@phpbits were there any other issues you noted when testing ClassifAI on WP 5.6 or just this warning in the block editor?

**Comment by barneyjeffries** [comment]
@jeffpaul I have set-up a local env and run the plugin on 5.8 with no issues. I also created a remote env and set-up the plugin there - image processing seems to be working as expected and the OCR is working. I ran the WP-CLI commands locally (apart from image related ones) and they worked correctly. The only thing I experienced which was remotely like an error was a 500 console error when I tried to run the Smart Crop in the media library - screenshot below. But this happened once and then ran successfully every time after that.

<img width="589" alt="Screenshot 2021-08-24 at 12 35 09" src="https://user-images.githubusercontent.com/20792906/130626518-a9019183-94e1-4831-9db3-c4add54de0ed.png">
